changelog/16246.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+secret/pki: Do not fail validation with a legacy key_bits default value when signing CSRs  
```

Comment:
Should probably clarify this is only for roles with `any` keytype?

Reply:
Done.